ehealthid-rp/src/test/java/com/oviva/ehealthid/relyingparty/MainTest.java

```diff
@@ -3,7 +3,9 @@
 import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
 import static io.restassured.RestAssured.*;
 import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import com.github.jknack.handlebars.internal.text.StringEscapeUtils;
```

Comment:
Ugh, do you really need to drag an interal utils of a dependency we don't want in for that one method?

Reply:
No I found a workaround :)